plugins/inventory/aws_ec2.py

```diff
@@ -51,6 +51,14 @@
               - Available filters are listed here U(http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html#options).
           type: dict
           default: {}
+        includes_entries_matching:
+          description:
+              - A list of filters. The instances matching at least one of the filters are included in the result.
```

Comment:
I needed to read this twice to parse it properly.  Oddly enough I found the changelog easier to understand than this description.

```suggestion
              - A list of filters. Any instances matching at least one of the filters are included in the result.
```

Reply:
:-), thank you for the feedback.